### ISSUE TITLE
Mouse gets away from ammeter after zoom

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#301]
Zoom out, then drag ammeter.  It is in wrong coordinate frame and drags at wrong speed depending on your screen size.

Thanks @Denz1994 !

**Comment by samreid** [comment]
It looks like the `startDrag` pattern of forwarding events doesn't support zoom too well.  @jonathanolson any suggestions?

**Comment by samreid** [comment]
Proposed fix is above, @Denz1994 can you please pull and verify?

**Comment by jonathanolson** [comment]
> It looks like the startDrag pattern of forwarding events doesn't support zoom too well. @jonathanolson any suggestions?

It seems the startDrag pattern in general is independent of anything like zoom. Is this a SimpleDragHandler issue, or specific to the zoom code you have? Let me know if consulting on it would help.

**Comment by samreid** [comment]
@jonathanolson it would be great to take a look when our time matches up.

**Comment by samreid** [comment]
@jonathanolson and I discussed it, and it seems like `targetNode` is the best solution for MovableDragHandler.  We tested the new `DragListener` implementation @jonathanolson has been implementing and demonstrated in http://localhost/scenery/examples/input.html and it works great for different coordinate frames.  I'll close this and if/when we upgrade sims to use the new DragListeners, we can revisit as a separate issue.
